test/distributed/cases/function/udf.sql

```diff
@@ -0,0 +1,22 @@
+-- @suite
+-- @setup
+
+-- @case
+-- @desc:test for create function (no arg)
+-- @label:bvt
+create function helloworld () returns int language sql as 'select id from test_table limit 1';
+
+-- @case
+-- @desc:test for create function (multiple args)
+-- @label:bvt
+create function twosum (x int, y int) returns int language sql as 'select $1 + $2' ;
+
+-- @case
+-- @desc:test for drop function (no arg)
+-- @label:bvt
+drop function helloWorld ();
+
+-- @case
+-- @desc:test for drop function (multiple args)
+-- @label:bvt
+drop function charAt (varchar, int);
```

Comment:
can you drop twosum with different param types?   what do we expect, fail or succeed?

Reply:
We expect this to be failed, as function is specified by both its name and parameters.